src/lib/components/workspace/members/Tile.svelte

```diff
@@ -0,0 +1,403 @@
+      isMemberDemotePopup = false;
+      notifications.success(`${user.name} is now a member`);
+    } else {
+      notifications.error(
+        `Failed to change role for ${user.name}. Please try again.`,
+      );
+    }
+  };
+  const handleMemberPromotePopUpSuccess = async () => {
+    const response = await teamServiceMethods.promoteToAdminAtTeam(
+      openTeam.teamId,
+      user.id,
+    );
+    if (response) {
+      teamRepositoryMethods.modifyTeam(openTeam.teamId, response);
+      await teamServiceMethods.refreshWorkspace(userId);
+      isMemberPromotePopup = false;
+      notifications.success(`${user.name} is now an admin`);
+    } else {
+      notifications.error(
+        `Failed to change role for ${user.name}. Please try again.`,
+      );
+    }
+  };
+
+  const handleMemberOwnershipPopUpSuccess = async () => {
+    const response = await teamServiceMethods.promoteToOwnerAtTeam(
+      openTeam.teamId,
+      user.id,
+    );
+    if (response) {
+      teamRepositoryMethods.modifyTeam(openTeam.teamId, response);
+      await teamServiceMethods.refreshWorkspace(userId);
+      isMemberOwnershipPopup = false;
+      notifications.success(
+        `${user.name} is now the new Owner of ${openTeam.name}.`,
+      );
+    } else {
+      notifications.error(
+        `Failed to update access of Owner. Please try again.`,
+      );
+    }
+  };
+</script>
+
+{#if isMemberRemovePopup}
+  <RemoveConfirmationPopup
+    title={`Remove user?`}
+    description={`<p style="font-size:12px;" class="text-textColor">
+      Are you sure you want to remove <span
+        class="text-whiteColor">"${user.name}"</span
+      >
+      ? They will lose access to the <span
+        class="text-whiteColor">"${openTeam?.name}"</span> team.
+    </p>`}
+    onSuccess={handleMemberPopUpSuccess}
+    onCancel={handleMemberPopUpCancel}
+  />
+{/if}
+
+{#if isMemberPromotePopup}
+  <MemberChangeRolePopup
+    title={`Changing Role?`}
+    teamName={openTeam?.name}
+    teamLogo={openTeam?.logo}
+    description={`
+    <div class="d-flex tile rounded mb-3">
+  <div
+    class="info d-flex align-items-center"
+  >
+    <div class="d-flex align-items-center justify-content-center" style="width: 36px;
+    border: 1px solid var(--border-color);
+    height: 36px;
+    border-radius: 50%;">
+      <span>${user.name[0].toUpperCase()}</span>
+    </div>
+    <div class="name px-2">
+      <span style="font-size:12px;" class="text-whiteColor">${
+        user.name
+      }</span><br />
+      <span style="font-size:12px;" class="text-textColor">${user.email}</span>
+    </div>
+  </div>
+</div>
+    
+    <p style="font-size:12px;" class="text-textColor">
+    You are assigning the role of an '<span class="text-whiteColor">Admin</span>' to ${
+      user.name
+    }. Following access will be provided to ${user.name}:</p>
+    <ul class="ps-4 text-textColor" style="font-size:12px;">
+      <li>Create New Workspaces.</li>
```

Comment:
Can we take this permission list from a constant file so that if any other permission is added/removed. We can do it over there.

Reply:
constant implemented